gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -1,42 +1,123 @@
 from panther_base_helpers import deep_get
 from panther_base_helpers import gsuite_parameter_lookup as param_lookup
 
+EXCLUDED_DOMAINS = ["runpanther.io"]
```

Comment:
This seems like a useful exclusion for us, but not our customers. Perhaps this should be `example.com` with a comment above it saying something like `# Change this to include any domains you want to trust shares with automatically`.

Also, this should be a set as well I think.

```suggestion
EXCLUDED_DOMAINS = {"example.com"}
```